library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -335,7 +583,8 @@ rocblas_status gemm_ex_handle_transpose(rocblas_handle    handle,
     static const bool arch_lt906 = handle->device_arch_id() < 906;
     const To* c_in;
     unsigned int ldi, stride_i;
-    if(!arch_lt906 && (std::is_same<Ti, float>{} || std::is_same<Ti, double>{}) &&
+
+    if(!arch_lt906 && (std::is_same<Ti, float>::value || std::is_same<Ti, double>::value) &&
```

Comment:
`std::is_same<T1,T2>{}` can be used as shorthand instead of `std::is_same<T1,T2>::value`, but this is not critical.

Reply:
understand I will correct it on third commit "update complex change".